site/doc/1.11/journal.md

```diff
@@ -356,6 +356,38 @@ when an amountless posting is balanced using a price's commodity, or
 when -V is used.) If you find this causing problems, set the desired
 format with a commodity directive.
 
+#### Amount Expressions
+
+An amount may also be comprised of multiple parts joined by basic
+arithmetic operators; in this case, each individual value follows the
+structure described above, while the joins allow:
+
+-   addition, subtraction, and multiplication using the standard ASCII
+    operators (`+`, `-`, `*`), either adjacent to the amount(s) or
+    separated by spaces
+-   parenthesised (sub-)expressions, to be evaluated before any
+    operations outside them. Note, however, that all commodity symbols
+    must be attached to an amount inside the parentheses: `($10 * 0.2)`
+    works, `$(10 * 0.2)` does not.
+-   multiple commodities in the same expression
+
+Multiplication of commodities is handled a bit differently than may be
+expected; if the amount to the right of the operator includes an
+explicit commodity, that commodity replaces any to the left for
+compatibility with [automated postings](#automated-postings). If that
+right value includes multiple commodities itself (parenthesised to obey
```

Comment:
I don't think we should discard commodities. Can we instead disallow multiplying by amounts, ie only multiplying by a number will be legal.

Reply:
> I looked into the transaction modifiers and found that this is exactly how they currently operate. It's undocumented behaviour

You're right! I'd rather enforce that the multiplier is a commodityless number for now. Quietly discarding information from one side of an expression is something we should avoid. I'll clarify this in docs. I agree that it should work the same in transaction modifiers and in amount expressions.

Reply:
First take of it at #913, pending further discussion.